spec/features/dams_object_spec.rb

```diff
@@ -434,6 +435,10 @@
       visit catalog_index_path( { q: 'THsdtk', sort: 'title_ssi asc' } )
       expect(page).to have_selector('h3', :text => "Image File Test")
     end
+    it 'should index fulltext of complex object CSV file' do
+      visit catalog_index_path( { q: 'Testing CSV', sort: 'title_ssi asc' } )
+      expect(page).to have_content("Image File Test")
```

Comment:
The name of the test object with the title 'Image File Test' is a little misleading now since we're using that object to test more than image files and associated behavior. Given that it's used in the context of 'viewing files' should we alter the name to something more generic?  (https://github.com/ucsdlib/damspas/blob/feature/fulltext_index/spec/features/dams_object_spec.rb#L385-L442)